### ISSUE TITLE
no torrent results

### ISSUE THREAD

**Issue by rlenferink** [isohuntto/openbay#127]
i have this script running on my website, but i can't search for torrents, it returns a 404 error
also when i go to 'browse torrents' no torrents will appear.

i use the remote database of isohunt

do you guys know how i can fix it ?

if you want a live view of the site, click [here](http://www.thepiratebay.agency/)

**Comment by TPBT-OFFICIAL** [comment]
Its probably just the remote SPHINX server offline don't worry if its not working in some hours then there may be a problem! turn debug mode on in /protected/config/debug.php

**Comment by rlenferink** [comment]
i have this problem for approximately 4 days right now.
turning debug mode on says:
- - - - - - -
CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect because the target machine actively refused it.
- - - - - - -
there is also a problem with the sphinx module. that's probably because there isn't a connection with the database.

first it worked all fine, didn't change anything, and it isn't working anymore.

**Comment by Lacedaemon** [comment]
Installed for the first time last night, and I too am having the same issue. Can anyone with their own database and Sphinx server confirm if theirs is working?

At first I thought I did something wrong, as I had permissions issues at the start, but now at least I know it isn't just me.

**Comment by nitmir** [comment]
Mine is working. As said on the sphinx conf page, you need a least sphinx 2.1.9, beware that the one packaged on debian/ubuntu is 2.0.4 and openbay won't work with it.
Moreover, you need to give to openbay conf the sphinx port corresponding to its sql interface (the one corresponding to the sphinx conf listen = 3306:mysql41, you probably want to use an other port (like 9306) if you are running mysql on the same server)

**Comment by rlenferink** [comment]
if i use the remote database, do i still have to setup sphinx?

**Comment by ekoice** [comment]
@Lacedaemon I got it working with my own sphinx and db, check http://vk.al/ but...dont know how to fix  the /browse section not showing results at all.
basically, on my website, it looks like this: http://i.imgur.com/4DIafLb.png
while I would like to look like: http://i.imgur.com/tOtaUL6.png
If someone would like to help...will be much appreciated.
thank you.

**Comment by nitmir** [comment]
@rlenferink nope, you don't have to, there also a remote sphinx instance. You just need sphinx if you want to use your own database.

**Comment by rlenferink** [comment]
alright, then i think the remote sphinx instance isn't working, because my recent torrents are working fine, but when i search something it isn't working

**Comment by MihaiB** [comment]
The remote Sphinx is currently down.

I've proposed a Vagrant setup in #135 using the remote MySQL but a local Sphinx. The entire set up is done with a few short scripts.

You can try it out in a local VM to see that it works then make the same Sphinx configuration on your servers.
